projects/control/multi_object_search/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenDR multi object search demo
+<div align="left">
+  <a href="https://opensource.org/licenses/Apache-2.0">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" height="20">
+  </a>
+</div>
+
+Live demo of mobile manipulation using the [OpenDR toolkit](https://opendr.eu).
+
+
+## Set-up
+Follow the setup described for the [multi_object_search tool](/docs/reference/multi_object_search.md). 
+
+## Running the example
+Mobile manipulation tasks in the analytical environment can be run as follows:
+```bash
+python multi_object_search_demo.py
+```
+
+The demo either executes a training or evaluation instance. By default it will evaluate all evaluation-scenes for 75 episodes sequentially.
```

Comment:
```suggestion
The demo either executes a training or evaluation instance. By default, it will evaluate all evaluation-scenes for 75 episodes sequentially.
```